package.json

```diff
@@ -47,13 +47,16 @@
     "postversion": "git push && git push --tags"
   },
   "dependencies": {
-    "@babel/core": "^7.13.15",
+    "@glimmer/component": "^1.0.4",
+    "@glimmer/tracking": "^1.0.4",
     "ember-cli-babel": "^7.26.3",
-    "ember-cli-htmlbars": "^5.7.1"
+    "ember-cli-htmlbars": "^5.7.1",
+    "ember-modifier": "^2.1.2"
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^2.2.5",
+    "@ember/test-waiters": "^3.0.0",
```

Comment:
Should be a normal dependency I think since it's used in `addon/file.js`?

Reply:
Right – fixed in 1ee1569